app/controllers/external_users/disc_evidence_coversheets_controller.rb

```diff
@@ -0,0 +1,52 @@
+class ExternalUsers::DiscEvidenceCoversheetsController < ExternalUsers::ApplicationController
+  skip_load_and_authorize_resource
+
+  def new
+    @disc_evidence_coversheet = DiscEvidenceCoversheet.new(claim_id: params[:claim_id])
+    authorize_coversheet!
+  end
+
+  def create
+    @disc_evidence_coversheet = DiscEvidenceCoversheet.new(disc_evidence_coversheet_params)
+    authorize_coversheet!
+
+    respond_to do |format|
+      format.html do
+        render 'pdfs/templates/disc_evidence_coversheet.html.erb'
+      end
+
+      format.pdf do
+        render(
+          pdf: "disc_evidence_coversheet_#{@disc_evidence_coversheet.case_number}",
+          margin: { top: 10, bottom: 10, left: 10, right: 10 },
+          template: 'pdfs/templates/disc_evidence_coversheet.html.erb',
+          disposition: 'inline'
+        )
```

Comment:
**Accessibility**: 
- Please add a document title using `title: 'Disc Evidence Coversheet'`
- Document language is missing, however, wicked_pdf doesn't seem to have an option to set this

Reply:
👍 done